packages/xstate-vue/test/UseSelector.vue

```diff
@@ -38,7 +38,7 @@ export default defineComponent({
 
     let rerenders = 0;
 
-    onRenderTracked(() => {
```

Comment:
I don't know why this worked previously or why the resulting value was `3`. 

From what I can tell (I don't know Vue), `onRenderTracked` is called only once for each reactive dependency:

> Registers a debug hook to be called when a reactive dependency has been tracked by the component's render effect.

Since there aren't 3 reactive dependencies, I'm not sure why the resulting value had been `3`.

`onRenderTriggered` appears to be what we want here:

> Registers a debug hook to be called when a reactive dependency triggers the component's render effect to be re-run.

And from that definition it appears that `2` is the correct value. It renders, we click "Increment", it re-renders due to `count` dep changing (1), we click "Increment" again, and it re-renders again due to `count` dep changing (2).